modules/core/src/main/scala/Tags.scala

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2019 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+
+object Tags {
+
+  // The software package, framework, library, or module that generated the associated Span. E.g., "grpc", "django", "JDBI".
+  def Component(c: String) = Field("component", c)
+
+  object Db {
+    private val prefix = "db"
+
+    // Database instance name.
+    // E.g., In java, if the jdbc.url="jdbc:mysql://127.0.0.1:3306/customers", the instance name is "customers".
+    def Instance(i: String) = Field(s"$prefix.instance", i)
+
+    // A database statement for the given database type.
+    // E.g., for db.type="sql", "SELECT * FROM wuser_table"; for db.type="redis", "SET mykey 'WuValue'".
+    def Statement = Field(s"$prefix.statement", _)
```

Comment:
The uppercase method names weird me out. I think I might prefer the inner objects and method names match up exactly with the tags they create, which means lowercasing the identifiers and adding underscores in a few places; i.e., `http.status_code(200)` would yield `"http.status_code" -> 200`. What do you think?